packages/devtools_app/macos/Runner.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 48;
+	objectVersion = 54;
```

Comment:
are these `macos/` changes intentional?

Reply:
Seems to be result of too fresh version of flutter. Reverted.